README.md

```diff
@@ -1,21 +1,44 @@
 Cypher Editor
 =============
 
-The goal of this project is to provide feature-rich support
-of [Cypher](http://www.opencypher.org/) query language in
-JavaScript-based editors. 
+<p align="center">Cypher-editor provide feature-rich support of Cypher query language in JavaScript-based editors</p>
 
-It includes, but not limited to functionality such as:
+<p align="center"><img src="https://cloud.githubusercontent.com/assets/24562497/25952124/6942acb6-3668-11e7-8323-05802debdefc.png" width=700 alt="Screenshot of Example Documentation created with Slate"></p>
```

Comment:
I like the picture. But we need to:
- Use some multi-line query (e.g. 5-8 lines) so it will be visile in screnshot
- Ensure that this query looks pretty (now on the screenshot there are whitespace issues)